Core/src/org/sleuthkit/autopsy/mainui/datamodel/BlackboardArtifactDAO.java

```diff
@@ -153,11 +154,11 @@
       
 
     TableData createTableData(BlackboardArtifact.Type artType, List<BlackboardArtifact> arts) throws TskCoreException, NoCurrentCaseException {
-        Map<Long, Map<BlackboardAttribute.Type, Object>> artifactAttributes = new HashMap<>();
+        Map<Long, Map<BlackboardAttribute.Type, Object>> artifactAttributes = new LinkedHashMap<>();
```

Comment:
Can you add a comment that we're using a linked hash map to preserve ordering?

Reply:
Sure.